crates/bevy_remote/src/lib.rs

```diff
@@ -153,7 +153,126 @@
 //! - `has`: A map associating each type name from `has` to a boolean value indicating whether or not the
 //!   entity has that component. If `has` was empty or omitted, this key will be omitted in the response.
 //!
+//! ### Example
+//! To query all entities and all of their Reflectable components (and retrive their values), you can pass
```

Comment:
Can we introduce the "normal" query syntax first? e.g. by querying for just a `Transform::type_path()`